### ISSUE TITLE
Error compiling tcpdump 4.9.3

### ISSUE THREAD

**Issue by gianogli** [ntop/PF_RING#663]
I'm using Ubuntu 16.04.7 LTS and PF_RING branch "dev".
If I run ./configure into userland/tcpdump-4.9.3 folder I obtain this error:
```[...]
checking for main in -lrpc... no
checking for library containing getrpcbynumber... none required
checking for local pcap library... ../libpcap-1.9.1/libpcap.a
checking for pcap-config... ../libpcap-1.9.1/pcap-config
checking for pcap_loop... no
configure: error: This is a bug, please follow the guidelines in CONTRIBUTING and include the
config.log file in your report.  If you have downloaded libpcap from
tcpdump.org, and built it yourself, please also include the config.log
file from the libpcap source directory, the Makefile from the libpcap
source directory, and the output of the make process for libpcap, as
this could be a problem with the libpcap that was built, and we will
not be able to determine why this is happening, and thus will not be
able to fix it, without that information, as we have not been able to
reproduce this problem ourselves.
```
In the config.log file I found this error:
```
configure:5401: checking for local pcap library
configure:5629: result: ../libpcap-1.9.1/libpcap.a
configure:5633: checking for pcap-config
configure:5651: found ../libpcap-1.9.1/pcap-config
configure:5663: result: ../libpcap-1.9.1/pcap-config
configure:5725: checking for pcap_loop
configure:5725: gcc -o conftest -g -O2   conftest.c ../libpcap-1.9.1/libpcap.a   -lrt -ldl -lnl-genl-3 -lnl-3   ../lib/libpfring.a -lpthread -lrt >&5
../lib/libpfring.a(pfring_utils.o): In function `pfring_thirdparty_lib_init':
pfring_utils.c:(.text+0x1b9f): undefined reference to `dlopen'
pfring_utils.c:(.text+0x1be4): undefined reference to `dlsym'
../lib/libpfring.a(pfring_ft_dpi.o): In function `ndpi_lib_init':
pfring_ft_dpi.c:(.text+0x1d): undefined reference to `dlopen'
pfring_ft_dpi.c:(.text+0x5c): undefined reference to `dlsym'
collect2: error: ld returned 1 exit status
```
If I simply copy the userland/tcpdump-4.9.2 folder from the stable branch into the dev branch all works fine and I can compile it without errors.
How can I solve the issue with version 4.9.3?

Do you need other info?

Thanks :-)

**Comment by cardigliano** [comment]
It seems -ldl is missing, I will fix it

**Comment by cardigliano** [comment]
Fixed in bc7d1df04dc21d32208252100cabe37418e84612
